packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -62,6 +62,7 @@ class TracerWrapper(object):
     def __new__(
         cls,
         disable_batch=False,
+        disable_requests_instrumentation=True,
```

Comment:
this seems highly specific, don't you think? I think I'd rather add something like the "instruments" parameter but that can be sort of a "block list" where any of the libraries in that parameter **won't** be instrumented.

Reply:
Block list sounds like a better idea. The reason I went specific was because the tracer is normally expected to instrument GenAI-related packages, but has `requests` as a special additional scenario. Block list would be broad enough for future developments as well.